plugins/CoreHome/templates/_donate.twig

```diff
@@ -37,7 +37,7 @@
                 title="{{ 'CoreHome_SubscribeAndBecomePiwikSupporter'|translate }}"/>
 			<a class="donate-spacer">{{ 'CoreHome_MakeOneTimeDonation'|translate }}</a>
             <a href="index.php?module=CoreHome&action=redirectToPaypal&idSite=1&cmd=_s-xclick&hosted_button_id=RPL23NJURMTFA&bb2_screener_=1357583494+83.233.186.82"
-               rel="noreferrer"  target="_blank" class="donate-one-time">{{ 'CoreHome_MakeOneTimeDonation'|translate }}</a>
+               rel="noopener" target="_blank" class="donate-one-time">{{ 'CoreHome_MakeOneTimeDonation'|translate }}</a>
```

Comment:
any reason noreferrer was removed here? is the redirectToPaypal removing the referrer? may be good to still have it.